assignments/prototypes.js

```diff
@@ -15,13 +15,29 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
-
+function GameObject (thing) {
+	this.createdAt = thing.createdAt;
+	this.name = thing.name;
+	this.dimensions = thing.dimensions;
+}
+GameObject.prototype.destroy = function() {
+	return `${this.name} was removed from the game in the most unpleasant way.`
```

Comment:
> [...] in the most unpleasant way.

Why not pleasantly, like gingerly removing pieces from a chess board? Why must it be so unpleasant? 😁